src/otx/v2/adapters/torch/mmengine/mmdet/modules/models/heads/custom_atss_head.py

```diff
@@ -0,0 +1,282 @@
+"""Custom ATSS head for OTX template."""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import torch
+from mmdet.models.dense_heads.atss_head import ATSSHead
+from mmdet.models.utils.misc import multi_apply
+from mmdet.registry import MODELS
+from mmdet.structures.bbox.bbox_overlaps import bbox_overlaps
+from mmdet.utils.dist_utils import reduce_mean
+from torch import Tensor
+
+if TYPE_CHECKING:
+    from mmdet.utils import InstanceList, OptInstanceList
+    from mmengine.structures import InstanceData
+
+from otx.v2.adapters.torch.mmengine.mmdet.modules.losses.cross_focal_loss import (
+    CrossSigmoidFocalLoss,
+)
+from otx.v2.adapters.torch.mmengine.mmdet.modules.models.heads.cross_dataset_detector_head import (
+    CrossDatasetDetectorHead,
+)
+
+EPS = 1e-12
+
+# pylint: disable=too-many-arguments, too-many-locals,
+
+
+@MODELS.register_module()
+class CustomATSSHead(CrossDatasetDetectorHead, ATSSHead):
+    """CustomATSSHead for OTX template."""
+
+    def __init__(
+        self,
+        *args,
+        bg_loss_weight: float = -1.0,
+        use_qfl: bool = False,
```

Comment:
What does `qfl` stand for? Quality focal loss? If yes, can we use `use_quality_focal_loss` instead?